src/test/java/org/broadinstitute/hellbender/MainTest.java

```diff
@@ -1,13 +1,65 @@
 package org.broadinstitute.hellbender;
 
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.TestProgramGroup;
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.junit.Assert;
 import org.testng.annotations.Test;
 
-public final class MainTest extends CommandLineProgramTest{
+import java.util.Collections;
+import java.util.List;
+
+public final class MainTest extends CommandLineProgramTest {
 
     @Test(expectedExceptions = UserException.class)
     public void testCommandNotFoundThrows(){
         this.runCommandLine(new String[]{"Brain"});
     }
+
+
+    /** Main class for testing a single CLP */
+    private static final class SingleClpMain extends Main {
+
+        private Class<? extends CommandLineProgram> clazz;
+
+        private SingleClpMain(final Class<? extends CommandLineProgram> clazz) {
+            this.clazz = clazz;
+        }
+
+        @Override
+        protected List<String> getPackageList() {
+            return Collections.emptyList();
+        }
+
+        @Override
+        protected List<Class<? extends CommandLineProgram>> getClassList() {
+            return Collections.singletonList(clazz);
+        }
+    }
+
+    @CommandLineProgramProperties(
+            programGroup = TestProgramGroup.class,
+            summary = "OmitFromCommanLine test",
+            oneLineSummary = "OmitFromCommanLine test",
+            omitFromCommandLine = true)
+    public static final class OmitFromCommanLineCLP extends CommandLineProgram {
+
+        @Override
+        protected Object doWork() {
+            return 1;
```

Comment:
Can you make "1" into a public class constant named "OMIT_CLP_RETURN_VALUE" or something similar here, and then use that value in the assert below to check for equality with that same constant.

Reply:
Done.